src/styles/main.css

```
@@ -0,0 +1,322 @@
+  background-color: var(--color-bluegray-100);
+  min-height: 100vh;
+}
+
+.visually-hidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.5.0)

[error] Unexpected deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)